### ISSUE TITLE
optimal_parameter : None

### ISSUE THREAD

**Issue by mak-rayate** [SAP-samples/hana-ml-samples#18]
Hello , 

I am doing hyperparameter tuning using unified classification. But I am getting optimal parameter as "None" in report. I tried both RandomSearchCV and GridSearchCV. Can you please tell me where I am doing wrong. 

rfc = UnifiedClassification(func = 'RandomDecisionTree')

gscv = GridSearchCV(estimator=rfc, 
                    param_grid={'n_estimators': [50,80,90,100,200,300,400,500,600,800],
                                'max_depth': [50, 60, 70, 80, 90, 100, 110,120,150],
                                'split_threshold': [0.1, 0.4, 0.7, 1],
                                'min_samples_leaf':[1,2,3,4]},
                    train_control=dict(fold_num=5,
                                       random_state=1,
                                       resampling_method='cv'), 
                    scoring='error_rate')

gscv.fit(data=hdf_train, key= 'id',
         label='label',
         features=features,
         partition_method='stratified',
         partition_random_state=1,
         stratified_column='label',
         build_report=True)
         
         
![image](https://user-images.githubusercontent.com/67750027/150069410-15cf0514-82b7-498c-b594-6c578527bbd0.png)

**Comment by raymondyao** [comment]
Random Decision Trees currently doesn't support hyper-parameter tuning. You may use HybridGradientBoostingTrees instead.
